### DISCUSSION TITLE
Please remove unused services for android like FcmInstanceIDService

### DISCUSSION THREAD

**Discussion by jimmymjing** [airsdk/apm#99] [General]
Hi, I just tried the latest version of APM, but the unused services like `FcmInstanceIDService` is still auto generated. I have to manually comment it out otherwise I won't be able to publish the Android APK... I think most of us would get tripped by this once...
```
<service android:name="com.distriqt.extension.pushnotifications.fcm.FcmInstanceIDService" android:exported="true">
      <intent-filter>
        <action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>
      </intent-filter>
    </service>
```

**Comment by marchbold** [comment]
Hi,

We are working on a fix for this in the push notifications ane. https://github.com/distriqt/ANE-PushNotifications/issues/520

Cheers
